html/income.html

```diff
@@ -234,6 +255,29 @@
         window.bridge = bridge
       })()
 
+      async function refresh() {
+        const content = document.querySelector(".spinnerContainer")
+
+            content.classList.remove("hidden")
+            let refreshedInfo
+            try {
+              refreshedInfo = await apiRequests.createRefreshTask()
+              document.getElementById("refreshMessage").classList.remove("hidden")
+              document.getElementById("refreshTime").innerHTML = new Date()
+            } catch (e) {
+              console.error(e)
+              content.classList.add("hidden")
+              return
+            }
+            content.classList.add("hidden")
+
+            if (!refreshedInfo.length) {
+              return
+            }
+
+            renderPayrollData(refreshedInfo)
+      }
+
```

Comment:
I know this is just example code but it might be nice to factor `refresh` and any other common elements out into a script, just for clarity and to make maintenance easier.  This `refresh` looks identical to the one in `employments.html`

Reply:
I had originally thought of that (it feels dirty as an engineer to have duplicate code I know) but because this is a quickstart it's actually better to do it this way. Clients are usually coming to get started with one product and it's better to have the code in one place as opposed to having them flip through files.